code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -61,17 +61,129 @@
 	embed_falloff_tile = -5
 	wound_falloff_tile = -2
-	shrapnel_type = /obj/item/stack/rods
+	shrapnel_type = /obj/item/ammo_casing/rebar
 
 /obj/projectile/bullet/rebarsyndie
 	name = "rebar"
 	icon_state = "rebar"
-	damage = 35
+	damage = 55
 	speed = 0.4
 	dismemberment = 2 //It's a budget sniper rifle.
 	armour_penetration = 20 //A bit better versus armor. Gets past anti laser armor or a vest, but doesnt wound proc on sec armor.
 	wound_bonus = 10
-	bare_wound_bonus = 10
+	bare_wound_bonus = 30
 	embedding = list(embed_chance=80, fall_chance=1, jostle_chance=3, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=3, jostle_pain_mult=2, rip_time=14)
 	embed_falloff_tile = -3
-	shrapnel_type = /obj/item/stack/rods
+	shrapnel_type = /obj/item/ammo_casing/rebar/syndie
+	var/modified = FALSE
+	var/obj/item/pen/pen = null
+
+/obj/projectile/bullet/rebar/zaukerite
+	name = "zaukerite shard"
+	icon_state = "rebar_zaukerite"
+	damage = 60
+	speed = 0.6
+	dismemberment = 2
+	damage_type = TOX
+	eyeblur = 5
+	armour_penetration = 30
+	wound_bonus = 0
+	bare_wound_bonus = 20
+	embedding = list(embed_chance=90, fall_chance=0, jostle_chance=5, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.8, pain_mult=6, jostle_pain_mult=2, rip_time=30)
+	embed_falloff_tile = -5
+	shrapnel_type = /obj/item/ammo_casing/rebar/zaukerite
+
+/obj/projectile/bullet/rebar/hydrogen
+	name = "metallic hydrogen bolt"
+	icon_state = "rebar_hydrogen"
+	damage = 40
+	speed = 0.4
+	dismemberment = 2
+	damage_type = BRUTE
+	armour_penetration = 16
+	wound_bonus = -15
+	bare_wound_bonus = 30
+	embedding = list(embed_chance = 70, fall_chance=1, jostle_chance=3, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.6, pain_mult=4, jostle_pain_mult=2, rip_time=18)
+	embed_falloff_tile = -3
+	shrapnel_type = /obj/item/ammo_casing/rebar/hydrogen
+
+/obj/projectile/bullet/rebar/healium
+	name = "healium bolt"
+	icon_state = "rebar_hydrogen"
+	damage = 0
+	speed = 0.4
+	dismemberment = 0
+	damage_type = BRUTE
+	armour_penetration = 100
+	wound_bonus = -100
+	bare_wound_bonus = -100
+	embedding = list(embed_chance = 0)
+	embed_falloff_tile = -3
+	shrapnel_type = /obj/item/ammo_casing/rebar/healium
+
+/obj/projectile/bullet/rebar/healium/on_hit(atom/target, blocked = 0, pierce_hit)
+	. = ..()
+	if(isliving(target))
+		var/mob/living/breather = target
+		breather.SetSleeping(3 SECONDS)
+		var/need_mob_update
+		need_mob_update = breather.adjustFireLoss(-30, updating_health = TRUE, required_bodytype = BODYTYPE_ORGANIC)
+		need_mob_update += breather.adjustToxLoss(-30, updating_health = TRUE, required_biotype = BODYTYPE_ORGANIC)
+		need_mob_update += breather.adjustBruteLoss(-30, updating_health = TRUE, required_bodytype = BODYTYPE_ORGANIC)
+		need_mob_update += breather.adjustOxyLoss(-30, updating_health = TRUE, required_biotype = BODYTYPE_ORGANIC, required_respiration_type = ALL)
```

Comment:
why +=? do you want like ||=?

Reply:
||= wouldn't adjust the overall health of the mob displayed in the health scanner and health doll.

Reply:
I do not understand how. we want need_mob_update to be "truthy" if we actually change anything, and "falsey" if not, yes? wouldn't ORing it with the output of the loss procs accomplish that fine

Reply:
Hm, earlier it didnt let the hp bar on HUDs update, but seems to work fine now. Must have been something else interfereing.